chrome.manifest

```diff
@@ -1,7 +0,0 @@
-content zoteroocr chrome/content/
```

Comment:
Please don't remove this file. It is required for a plugin which supports both Zotero 6 and Zotero 7.

Reply:
As discussed in the issue https://github.com/UB-Mannheim/zotero-ocr/issues/52#issuecomment-1975652990 , I am not interested in supporting Zotero 6 and I prefer to submit a Z7-only version on this side branch.